Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine AS builder
+
+RUN apk update && \
+    apk --no-cache --update add build-base
+
+RUN apk add git automake autoconf libtool unbound-dev
+
+COPY . /
+
+RUN ./autogen.sh && ./configure && make
+
+FROM alpine:latest
+
+RUN apk update && apk upgrade 
```

Comment:
Typically I:
1. call the first chuck "BASE"
2. Do an apk "update --no-cache [runtime deps]"
3. Then I do "from base as builder" so the discarded building phase uses the updates from the base
4. Then the final chunck is "from base" and already has the runtime deps

This make the overall build as fast and small as possible.
 =]